lib/indexer.rb

```diff
@@ -0,0 +1,151 @@
+module Argo
+  class SimpleLogJob
+    def initialize(log_val)
+      @log_val = log_val
+    end
+
+    def perform
+      sleep(rand 10)
+
+      log_filename = 'tmp/simple_job.log'
+      File.open(log_filename, 'a') { |log|
+        log.puts "#{@log_val}"
+      }
+    end
+  end
+
+  # TODO: DRY up the repetition btwn here and dor_controller
+  class Indexer
+    @@index_logger = nil
+    def self.index_logger
+      return @@index_logger if @@index_logger
+
+      @@index_logger ||= Logger.new("#{Rails.root}/log/indexer.log", 10, 3240000)
+      @@index_logger.formatter = proc do |severity, datetime, progname, msg|
```

Comment:
Unused block argument - `severity`. If it's necessary, use `_` or `_severity` as an argument name to indicate that it won't be used.<br>Unused block argument - `progname`. If it's necessary, use `_` or `_progname` as an argument name to indicate that it won't be used.